test/check-machines-nics

```diff
@@ -411,6 +411,28 @@ class TestMachinesNICs(VirtualMachinesCase):
                 f"#vm-subVmTest1-network-{self.nic_num}-mac",
                 self.mac if self.mac and self.vm_state == "shut off" else self.mac_current
             )
+            if self.vm_state == "running":
+                if self.source_type is not None and self.source_type != self.source_type_current:
+                    b.wait_visible(f"#vm-subVmTest1-network-{self.nic_num}-type-tooltip")
+                else:
+                    b.wait_not_present(f"#vm-subVmTest1-network-{self.nic_num}-type-tooltip")
+
+                if self.source is not None and self.source != self.source_current:
+                    b.wait_visible(f"#vm-subVmTest1-network-{self.nic_num}-source-tooltip")
+                # Changing source_type may in effect also change source, so tooltip being present is correct
+                # Only changing source type beteen bridge and direct should not affect source
```

Comment:
What makes this direct/bridge type transition special? Could there be other combinations where this is the case?

Reply:
On system connection where this test runs, currently you can only choose 3 types of virtual network interface: `Bridge to LAN, Direct attachement and Virtual Network.`
`Bridge to LAN` and `Direct attachemen`t share the same source set, therefore it's possible to change interface type between the two without source changing:
![Screenshot from 2022-03-07 11-02-32](https://user-images.githubusercontent.com/42733240/157009884-dad3f603-aa73-4a84-9289-eaf5fadd6f77.png)
![Screenshot from 2022-03-07 10-57-56](https://user-images.githubusercontent.com/42733240/157009886-c6c9ac64-5788-4a80-afa8-9f512314d947.png)

However virtual network has a difference set of sources, therefore when you switch to or from `virtual network`, source will reset and will always be different than in `bridge` or `direct attachement`
![Screenshot from 2022-03-07 10-58-48](https://user-images.githubusercontent.com/42733240/157010046-23efdb42-c6da-48f5-b3d4-6bd46c493083.png)

Reply:
Thanks for the detailed explanation!